File: Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/features/settings/model/view/HeaderSetting.java
// SPDX-License-Identifier: GPL-2.0-or-later

package org.dolphinemu.dolphinemu.features.settings.model.view;

import android.content.Context;

import org.dolphinemu.dolphinemu.features.settings.model.AbstractSetting;

public class HeaderSetting extends SettingsItem
{
  public HeaderSetting(Context context, int titleId, int descriptionId)
  {
    super(context, titleId, descriptionId);
  }

  @Override
  public int getType()
  {
    return SettingsItem.TYPE_HEADER;
  }

  @Override
  public AbstractSetting getSetting()
  {
    return null;
  }
}

File: Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/features/settings/model/view/HeaderSetting.kt
// SPDX-License-Identifier: GPL-2.0-or-later

package org.dolphinemu.dolphinemu.features.settings.model.view

import android.content.Context
import org.dolphinemu.dolphinemu.features.settings.model.AbstractSetting

open class HeaderSetting : SettingsItem {
    override val setting: AbstractSetting? = null

    constructor(
        context: Context,
        titleId: Int,
        descriptionId: Int
    ) : super(context, titleId, descriptionId)

    constructor(title: CharSequence, description: CharSequence?) : super(title, description)

    override val type: Int = TYPE_HEADER
}
